### PR TITLE
Add defaults (as comments) to Python.Runtime

### PR DESCRIPTION
**Pull request by aarjaneiro**
#### Description

Added default dllmap locations which can be uncommented and filled at /Launcher/Python.Runtime.dll.config. Further, added  instructions to the Algorithm.Python readme on how to specify the location for the Python 36 shared library. There is currently no _non-trivial_ way, it seems, to have default locations set for the user by probing a user's conda /lib/ folders without changes to the runtime itself.

Apologies for not being able to simply edit the last (closed) PR on account of the original branch being deleted.
#### Related Issue

A template to the solution of https://www.quantconnect.com/forum/discussion/8157/lean-install-with-python-on-linux-not-finding-python3-6m/p1/comment-26460 by adding defaults for non-windows OSes.

https://github.com/QuantConnect/Lean/issues/4902

#### Motivation and Context

If one has a non-standard installation, DllNotFound-type errors for python3.6m should be mitigated if following the template configs.

#### How Has This Been Tested?

Built and ran both a Python and C sharp algorithm on Linux (5.8.16-2-MANJARO; Mono JIT compiler version 6.13.0) without errors.

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [ ] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Launcher/Python.Runtime.dll.config [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+    <!-- For Linux and Mac users: uncomment and fill either of the following: -->
+    <!-- <dllmap os="linux" dll="python3.6m" target = "/home/{your_user_name}/miniconda3/envs/{qc_environment}/lib/libpython3.6m.so"/> -->
+    <!-- <dllmap os="osx" dll="python3.6m" target = "/Users/{your_user_name}/anaconda3/lib/libpython3.6m.dylib"/> -->
+</configuration>
+
+
+
+
+
+
```
Comment (Martin-Molinero) [inline Launcher/Python.Runtime.dll.config:12]: nit - unrequired new lines

**Review by Martin-Molinero — APPROVED** [review]
Thanks! Just a minor comment

File: Launcher/Python.Runtime.dll.config (hunk 1 above)
Comment (aarjaneiro) [reply Launcher/Python.Runtime.dll.config:12]: My bad!

Comment (Martin-Molinero) [reply Launcher/Python.Runtime.dll.config:12]: > My bad!

No worries 👍 let's please remove them though

**Pull request merged**